modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,148 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have application client credentials residing outside of Anypoint Platform if you were previously using a different API management platform. You must first migrate those client applications to Anypoint Platform, with their client ID and client secret credentials, before you can start using Anypoint Platform.
```

Comment:
```suggestion
You might have application client credentials residing outside of Anypoint Platform if you were previously using a different API management solution. You must first migrate those client applications to Anypoint Platform, with their client ID and client secret credentials, before you can start using Anypoint Platform.
```
Small fix: is not necessary a platform. 

> before you can start using Anypoint Platform.

This part is not fully true. You can use many other things, but definitively not those credentials as part of anypoint platform.

Reply:
Done! Thanks!